tests/unit/sagemaker/workflow/test_pipeline.py

```diff
@@ -200,10 +238,12 @@ def test_pipeline_update_with_parallelism_config(sagemaker_session_mock, role_ar
         name="MyPipeline",
         parameters=[],
         steps=[],
-        pipeline_experiment_config=ParallelismConfiguration(max_parallel_execution_steps=10),
```

Comment:
Was this just incorrect?

Reply:
You mean the usage of ParallelismConfiguration? Yea, that won't pass the pipeline definition generation for some reason. Will update

Reply:
Yea, passing that to the `pipeline_experiment_config` in the constructor. Seemed like the incorrect usage originally -- it needs to be passed to `create(parallelism_config=)` call like you already added. Was just checking.

Reply:
Ohh, get your point. Yea, the test for parallelism_config was not well-maintained.
Besides of this, the usage of `ParallelismConfiguration` does not work during pipeline definition generation. At this moment, customers and public docs are using dict define the config here. So I just updated accordingly.